src/routes/index.tsx

```diff
@@ -27,6 +27,11 @@ export default () => {
           <Route exact path={ROUTES.explore}>
             <Explore />
           </Route>
+
+          <Route exact path={ROUTES.adminPanel}>
+            <p>Admin panel will be done in next task</p>
```

Comment:
I think we should use a better text for this... Maybe `Coming soon`?

Reply:
Yeah, it could be but it's a temporary :)